tasks/failover_ip.yml

```diff
@@ -0,0 +1,21 @@
+- name: Add failover network interface alias
+  template:
+    src=network/interfaces.d/failover-ip
+    dest="{{ COMMON_FAILOVER_INTERFACE_CONFIG }}"
+    owner=root
+    group=root
+    mode=0644
+
+- name: Ensure failover network is included in configuration file
+  lineinfile:
+    dest=/etc/network/interfaces
+    state=present
+    line="source {{ COMMON_FAILOVER_INTERFACE_CONFIG }}"
+
+- name: Restart networking service
+  service:
+    name: networking
+    state: restarted
+
+- name: Bring up new interface
+  shell: "ifup {{ COMMON_FAILOVER_INTERFACE_NAME }}"
```

Comment:
The `command` module is preferred when no shell is needed.  (It will run the command directly, without and intermediary shell.)